pkg/util.go

```diff
@@ -132,3 +132,16 @@ func GenerateRandomID() int {
 	id := rand.Intn(1000000000) // Adjust the range as needed
 	return id
 }
+
+func MakeCurlCommand(method string, url string, header map[string]string, body string) string {
```

Comment:
Add checks for param and type of methods . Like if there is a body then only you need to add `--body` etc . These kind of checks.

Reply:
@charankamarapu I have added a body check in makeCurlCommand like this :
![image](https://github.com/keploy/keploy/assets/91385321/f0c3d263-bbab-4185-a813-777cc7ab0a65)
any invalid method type would result in 400 error and it won't make a .yaml file.

Reply:
Ok got it.